backend/web/migrations/0002_auto_20210331_0822.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 3.1.7 on 2021-03-31 08:22
```

Comment:
this file probably should not be in this PR
please do not forget to check your's PR content